mhs/common/mhs_common/workflow/sync_async.py

```diff
@@ -10,27 +10,31 @@
 from mhs_common.state import work_description as wd
 from mhs_common.transmission import transmission_adaptor as ta
 from mhs_common.workflow import common_synchronous
+from mhs_common.state import persistence_adaptor as pa
 
 logger = log.IntegrationAdaptorsLogger('MHS_SYNC_ASYNC_WORKFLOW')
 
-
 ASYNC_RESPONSE_EXPECTED = 'async_response_expected'
+MESSAGE_DATA = 'data'
+CORRELATION_ID = 'correlation_id'
 
 
 class SyncAsyncWorkflow(common_synchronous.CommonSynchronousWorkflow):
     """Handles the workflow for the sync-async messaging pattern."""
 
-    # TODO: This used to take an outbound transmission object
-    def __init__(self, transmission: ta.TransmissionAdaptor, party_id: str):
-
+    def __init__(self,
+                 party_id: str,
+                 transmission: ta.TransmissionAdaptor = None,
+                 sync_async_store: pa.PersistenceAdaptor = None):
         """Create a new SyncAsyncWorkflow that uses the specified dependencies to load config, build a message and
         send it.
-
         :param transmission: The component that can be used to send messages.
```

Comment:
Even when working on the outbound workflow, I don't think this transmission adaptor will be needed here, as the sync-async workflow itself doesn't make outbound requests, the other async workflows do that.

Reply:
I considered removing the references to the transmission but since the current sync-async outbound relies on it I think I will leave it as to avoid removing code/tests that might be needed/useful for the outbound ticket